editor/editor_settings.cpp

```diff
@@ -630,7 +630,8 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	/* Editors */
 
 	// GridMap
-	_initial_set("editors/grid_map/pick_distance", 5000.0);
+	// Range matches the SpinBox in the GridMap editor plugin.
+	EDITOR_SETTING(Variant::FLOAT, PROPERTY_HINT_RANGE, "editors/grid_map/pick_distance", 5000.0, "500,10000,1")
```

Comment:
Checked and I'm confident this is meters though.
```suggestion
	EDITOR_SETTING(Variant::FLOAT, PROPERTY_HINT_RANGE, "editors/grid_map/pick_distance", 5000.0, "500,10000,1,suffix:m")
```